resources/zib-ContactPerson.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-ContactPerson" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactPerson" />
+  <name value="ZibContactPerson" />
+  <title value="zib ContactPerson" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A contact is a  person not being a healthcare professional who is involved in the patient’s care, such as family members, caregivers, mental caretakers, guardians and legal representatives.&#xD;&#xA;&#xD;&#xA;In addition to identification information such as the name, address and contact information can also be entered. The relationship to the patient and the role this person has can be entered as well." />
+  <purpose value="This RelatedPerson resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) ContactPerson v3.4 (2020)](https://zibs.nl/wiki/ContactPerson-v3.4(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-contactperson-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/ContactPerson-v3.4(2020EN)" />
+    <name value="zib ContactPerson-v3.4(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="RelatedPerson" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="RelatedPerson">
+      <path value="RelatedPerson" />
+      <short value="Contact" />
+      <comment value="For the RelatedPerson.telecom and RelatedPerson.name element, we use the cardinality of FHIR. In the contact information zib there is a container that does not exist in FHIR. The container itself has a cardinality of 0 .. *. For the &#xD;&#xA;&#xD;&#xA;RelatedPerson.name element we use the same cardinality, because the given name is not allowed in this element. Some patients has more then 1 name, so the cardinality of 0 .. * it is possible to define multiple names." />
```

Comment:
Wellicht kan deze beagrumentatie over waarom we qua kardinaliteit afwijken beter op de de elementen zelf worden gezet.  

Wat betreft name: 
>Some patients has more then 1 name, so the cardinality of 0 .. * it is possible to define multiple names
Dit klopt niet. Vanwege het HumanName.use element kan het voorkomen dat er meerdere instanances van HumanName voorkomen per use. Hierdoor kunnen we de 0..1 niet overnemen. De zib kent het .use element niet.